### PR TITLE
Add signature workflows docs

### PR DESCRIPTION
**Pull request by gerrod3**
[noissue]

### CODE REVIEW

**Review by mdellweg — APPROVED** [review]
Left one suggestion for readability. Would merge either way.

File: docs/workflows/signature.rst [hunk 1]
```diff
@@ -0,0 +1,109 @@
+Signature Workflows
+===================
+
+``pulp_ansible`` supports Collection signing, syncing, and uploading. Collection signing adds extra
+validation when installing Collections with `ansible-galaxy` Check out the workflows below to see
```
Comment (goosemania) [inline docs/workflows/signature.rst:5]:
```suggestion
validation when installing Collections with `ansible-galaxy`. Check out the workflows below to see
```

File: docs/workflows/signature.rst [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+In order to verify signature validity on uploads you will need to store your trusted keyring on
+your Pulp system and set the ``pulp_ansible`` setting ``ANSIBLE_CERTS_DIR`` to the folder of that
+keyring. By default it is set to ``/etc/pulp/certs/`` if you want an easy place to store your
+keyring.
+
+.. note::
+    You can upload signatures without supplying Pulp your keyring, but ``pulp_ansible`` will not
+    perform validity checks on the uploaded signature.
+
+In order to have pulp_ansible sign collections stored in your repositories you will need to set up
```
Comment (goosemania) [inline docs/workflows/signature.rst:19]:
```suggestion
In order to have ``pulp_ansible`` sign collections stored in your repositories you will need to set up
```

File: docs/workflows/signature.rst [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+.. code-block:: bash
+
+    pulpcore-manager add-signing-service ansible-signing-service $SCRIPT_LOCATION $PUBKEY_FINGERPRINT
+
+Reference: `Signing Service <https://docs.pulpproject.org/pulpcore/workflows/signed-metadata.html>`_
+
+Signing Collections
+-------------------
+
+Sign collections stored in repository ``foo`` with the signing service ``ansible-signing-service``
```
Comment (goosemania) [inline docs/workflows/signature.rst:53]:
```suggestion
Sign collections stored in repository ``foo`` with the signing service ``ansible-signing-service``:
```

**Review by goosemania — COMMENTED** [review]
Nice docs!

**Pull request merged**